### PR TITLE
Added minitube profile

### PR DESCRIPTION
**Pull request by kortewegdevries**
Request: # 1139

This needs to be merged before I push profiles for apps from the same author. They all have common directory `Flavio Tordini`

### CODE REVIEW

File: etc/profile-m-z/minitube.profile [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+whitelist /usr/share/minitube
+include whitelist-common.inc 
+include whitelist-runuser-common.inc 
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+netfilter
+no3d
```
Comment (rusty-snake) [inline etc/profile-m-z/minitube.profile:41]: "Disable 3D hardware acceleration.", could cause performance issues.

File: etc/profile-m-z/minitube.profile [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+nodvd
+nogroups
+nonewprivs
+noroot
+notv
+nou2f
```
Comment (rusty-snake) [inline etc/profile-m-z/minitube.profile:47]: `novideo`?

Comment (kortewegdevries) [reply etc/profile-m-z/minitube.profile:47]: Wouldn't video break with it?

File: etc/profile-m-z/minitube.profile (hunk 1 above)
Comment (kortewegdevries) [reply etc/profile-m-z/minitube.profile:41]: Yeah,even without it I get Mesa terminal spams, I added it at the last moment. I'll remove it.

File: etc/profile-m-z/minitube.profile (hunk 2 above)
Comment (rusty-snake) [reply etc/profile-m-z/minitube.profile:47]: I thought it is to watch and don't need a webcam.

Comment (kortewegdevries) [reply etc/profile-m-z/minitube.profile:46]: I mixed up the two,pushed.

**Comment by kortewegdevries** [comment]
Off-topic: Is there a way to monitor dbus accesses of a program with something, other than, filtering `dbus-monitor` or flatpak`?

**Comment by rusty-snake** [comment]
> Is there a way to monitor dbus accesses of a program with something, other than, filtering dbus-monitor or 'flatpak'?

If the policy is `filter`, you can use `--dbus-log`. I plan to write a python-script which can parse the output and generate rules.

> Also how do other distros like Arch,SUSE handle pop-up notifications,pretty sure notify-send is common across Ubuntu (atleast Xubuntu),

`notify-send` exists also in Fedora, Arch, …
If there would be own standards they would be part of a DE not a distro. 

> do we allow pop-ups?

`org.freedesktop.Notifications` can be used to escape the sandbox. I should be avoided on sandboxed with high-attack-surface / strict profile / whitelist profile. If it is essential for a program or the program has "crash" as "error-handling" it must be allowed.

**Comment by kortewegdevries** [comment]
Thanks.

**Comment by rusty-snake** [comment]
Not yet complete https://github.com/netblue30/firejail/wiki/Restrict-D-Bus

**Comment by rusty-snake** [comment]
`org.freedesktop.Notifications`:
- sway: safe (AFAIK)
- gnome <= 3.36.0: unsafe
- gnome >= 3.36.1: safe
- kde: unsafe

**Comment by kortewegdevries** [comment]
Idk why I think screensaver may be used to steal user credentials since it the login screen is one of the places to snoop on and be sure that a password is entered. Is this possible? https://gitlab.gnome.org/GNOME/gnome-keyring/-/issues/5

**Comment by rusty-snake** [comment]
Merged, Thanks.

**Pull request merged**